content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
 
-### Site configuration
+```bash
+hugo mod init github.com/my-username/my-hugo-theme
+```
 
-You have a `config.toml` file in the `exampleSite` folder. Which gives you the ability to configure your site, such as `changing logo` or `customize menu`.
+After that you will find a file `go.mod` in your root directory. Add this file to your repository.
 
-#### Default configuration
+Your website is now ready to receive modules. Set up the theme as module in your configuration:
 
-Default configuration is those which come with the Hugo library. Remember that, all configuration does not come with every theme.
+```toml
+[[module]]
+[[module.imports]]
+path = "github.com/themeauthor/themename"
+```
 
- * `baseURL` defines your site URL. Give your site URL here (i.e. https://examplesite.org).
+{{% notice warning %}}Do this change in your config.toml file ***after*** you copy the exampleSite into your root directory (see the next step). {{% / notice %}}
 
- * `title` is for your site title. Which shown on the browser tab.
+The next time you run Hugo it will download the theme. You will not see that theme in your `themes` folder, because Hugo caches it in a temporary directory. But no need to worry, it will find the theme files.
 
- * `theme` name is already provided with every theme. don't need to customize this theme name.
+You will find a file called `go.sum` in your root directory. Add this file to your repository.
 
- * `summaryLeangth` is for the post excerpt limit. you can set your post excerpt length as you want to show.
+### Step 5: Copy the sample site
 
- * `paginate` set your pagination for page items. Such as blog posts, portfolio items, etc. If you set `paginate = 10` then it will show 10 posts on a page.
+Every theme comes with a folder named `exampleSite`. Copy the folders and files inside of that folder into your root directory. Be careful not to overwrite anything you already changed, but expect to override some files and folders that were created by step 2 above.
 
- * `googleAnalytics` is for activating analytics on your site. If you want google analytics to activate, then simply give your analytics ID here (i.e. `googleAnalytics= UA-123-45`). Or if you don't, then leave it empty.
+### Step 6: Add files to the repository
 
- * `disqusShortname` is for activating comment section on a blog page or other page which needs a comment section. get your Disqus short name from [here](https://disqus.com/). if you don't need the comment section, then leave it empty.
+Now your site is ready to run. Add a file with the name `.gitignore` to your root directory and fill it with the contents of [this autogenerated sample file](https://www.toptal.com/developers/gitignore/api/node,go,hugo). This will exempt several files from being added to the repository that are created automatically. You can change the list of supported programming languages and IDEs [at this link](https://www.toptal.com/developers/gitignore?templates=node,go,hugo).
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)